printer_mods/README.md

```diff
@@ -86,7 +86,8 @@ like so:
 |claytono|[Fysetc S6 V2 Mount](./claytono/fysetc-s6-v2-mount)| DIN Mount for the Fysetc S6 V2 controller |:x:|:heavy_check_mark:|:heavy_check_mark:|:x:|
 ||[Voron 2.4 Extrusion Jig](./claytono/2.4-extrusion-jig)|A jig help align extrusions to get a properly square frame |:x:|:x:|:heavy_check_mark:|:x:|
 ||[V0 Zip Tie Skirts](./claytono/v0-zip-tie-skirts)|V0 skirts with small loops added along the top to allow zip tyiing cables to them|:heavy_check_mark:|:x:|:x:|:x:
-| cmrny | [V0.1 Motor Panel Modification](./cmrny/v01_motor_panel_no_rub) | V0.1 Motor panel modification to keep umbilical and bowden tube from rubbing against motor belts | :heavy_check_mark: | :x: | :x: | :x: | :x:|`
+| cmrny | [V0.1 Motor Panel Modification](./cmrny/v01_motor_panel_no_rub) | V0.1 Motor panel modification to keep umbilical and bowden tube from rubbing against motor belts | :heavy_check_mark: | :x: | :x: | :x: | :x:|
+|| [V0.1 LGX Lite Mount for Mini AfterSherpa](./cmrny/v01_LGXLiteMount_MiniAfterSherpa) | V0.1 LGX Lite Mount for Mini AfterSherpa | :heavy_check_mark: | :x: | :x: | :x: | :x:|
```

Comment:
I would put the LGX Lite mod above Motor panel one, it will respect the alphabetical ordering.
And i's missing the entry for the V0.1 belt equalizer